src/pages/Privacy.vue

```diff
@@ -0,0 +1,130 @@
+
+                <section>
+
+                  <h1>Rosie – the Rose Voucher Recording System for Traders</h1>
+
+                  <h3>1. Privacy Policy</h3>
+
+                  <p>The Rose Voucher Recording System for Traders (hereby referred to as ‘Rosie’) is a Progressive Web Application (PWA) – for more information about PWAs, please see <a href="https://developers.google.com/web/progressive-web-apps" class="info" target="_blank">https://developers.google.com/web/progressive-web-apps/</a></p>
+
+                  <p>Rosie is owned by Alexandra Rose Charity (referred to here as ARC or “we”) whose registered office is Suite 5B, The Oast House, 5 Mead Lane, Farnham, Surrey, GU9 7DY.  ARC is registered with the Charity Commission for England &amp; Wales – Charity Number 211535.</p>
+
+                  <p>ARC takes your privacy very seriously. The policies and procedures we have in place are to protect the integrity and confidentiality of personal information, to comply with the Data Protection Act 1998 and other UK and EU regulations.</p>
+
+                  <p>Within the Data Protection Act 1998, Personal Data is defined as data that relates to a living individual who can be identified:</p>
+
+                  <ul>
+                      <li>from the data, or</li>
+                      <li>from the data and other information which is in the possession of, or likely to come into the possession of, the Data Controller, and includes any expression of opinion about the individual, and/or any indication of the intentions of the Data Controller or any other person in respect of the individual.</li>
+                  </ul>
+
+                  <p>By providing us with your Personal Data when using Rosie, or setting up an account to use Rosie, you give your consent that all Personal Data that you submit may be processed by us in the manner and for the purposes described below.  This Privacy Policy also forms part of our Terms of Use.</p>
+
+                  <p>Rosie has three user-categories to which this Privacy Policy applies:</p>
+
+                  <ul>
+                      <li>Trader - People or Organisations which operate market stalls or other retail outlets</li>
+                      <li>User - People who can log in to the site &amp; are authorised to use it on behalf of Traders</li>
+                      <li>Admin - staff from ARC and its technology partners who will support Rosie</li>
+                  </ul>
+
+                  <h3>How and Why We Collect Personal Data</h3>
+
+                  <p>Signing up for a Rosie account requires a user to provide certain basic information to set up &amp; operate the account.</p>
+
+                  <ul>
+                      <li>Name</li>
+                      <li>Business name</li>
+                      <li>Email address</li>
+                      <li>Password</li>
+                      <li>Market location</li>
+                      <li>Telephone number</li>
```

Comment:
We don't keep telephone numbers but they might. Still don't think it applies to The app though because ARC also keep payment details and they aren't mentioned.